presto-docs/src/main/sphinx/release/release-0.188.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.188
+=============
+
+General Changes
+---------------
+
+* Fix exception for illegal inputs to array :func:`slice` function.
+* Fix inverted sign for time zones ``Etc/GMT-12``, ``Etc/GMT-11``, ..., ``Etc/GMT-1``,
+  ``Etc/GMT+1``, ... ``Etc/GMT+12``.
+* Add environment support to db-backed resource groups manager.
+  Previously, configurations of different clusters have to be stored in separate databases.
+  With this change different cluster configurations can be stored in the same table and
+  Presto will use the new `environment` column to differentiate them.
+  The `node.environment` config property will be used to determine the current environment,
+  and Presto will load the configuration for that environment on startup.
+* Add `pages-index.eager-compaction-enabled` config property.
```

Comment:
```
Reduce GC spikes by compacting join memory over time instead of all at once
when memory is low. This can increase reliability at the cost of additional
CPU. This can be enabled via the ``pages-index.eager-compaction-enabled``
config property.